web/tailwind.css

```diff
@@ -130,6 +132,8 @@
   --text-800: #d1d5db;
   --text-900: #ffffff;
 
+  --chat-italics: #a3a3a3;
```

Comment:
Safest to use the `--text-N` variables here to ensure light mode is still readable.